src/linagora.esn.unifiedinbox/app/directives/main.spec.js

```diff
@@ -358,7 +358,7 @@ describe('The linagora.esn.unifiedinbox Main module directives', function() {
       });
 
       it('should be droppable element', function() {
-        expect(element.attr('esn-droppable')).to.equal('esn-droppable');
+        expect(element.attr('esn-droppable')).to.exist;
```

Comment:
You changed a test case logic to make it passed, which is not a good thing
If it exists, what is the value of if? If its value is not 'esn-droppable' anymore, we should only expect it to equal the new value.

Reply:
Same to above expect statements

Reply:
Yes I figured someone would ask about this. It is due to the way Webpack's PUG loader transpiles the PUG templates is different from before. `esn-droppable` is an attribute directive, and [it doesn't need to take any value](https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/blob/e60774c8e1d66cacc5202fda512d950fc096f255/src/frontend/js/modules/dragndrop.js#L63), so all of these work the same: 

1. `<example esn-droppable></example>`
2. `<example esn-droppable="esn-droppable"></example>`
3. `<example esn-droppable="whatever"></example>`

You can take a look at a usage example here: https://github.com/OpenPaaS-Suite/esn-frontend-inbox/blob/48432fe893a1ae0c78bf713ce5781129afbdf2e0/src/linagora.esn.unifiedinbox/views/sidebar/email/menu-item.pug#L4

Reply:
So the old test logic is actually bad because all that is necessary is for the `esn-droppable` to exist. It doesn't need to take a value. But it passed thanks to the old PUG backend loader.